macros/src/codegen/pre_init.rs

```diff
@@ -43,6 +43,13 @@ pub fn codegen(
         let mut core: rtic::export::Peripherals = core::mem::transmute(());
     ));
 
+    if app.args.cores == 1 {
+        stmts.push(quote!(
+            // To set the variable in cortex_m so the peripherals cannot be taken multiple times
+            let _ = cortex_m::Peripherals::steal();
```

Comment:
This caused https://github.com/rtic-rs/cortex-m-rtic/issues/351, it should probably read something like:

`let _ = rtic::export::Peripherals::steal();`

Reply:
This looks correct, I'll make a fix